### DISCUSSION TITLE
Change axis range of the plot

### DISCUSSION THREAD

**Discussion by Tomoya-Iizawa** [scikit-hep/cabinetry#457] [Q&A] [answered]
Hi,

Is there any option to change axis range in cabinetry.visualize.data_mc, especially for the ratio plot?

I am creating plots by cabinetry.visualize.data_mc, and then main distribution and data/MC ratio plot are plotted. The default y-axis raige seems [0.5, 1.5], is it possible to change the range e.g. [0.75, 1.25] or is it necessary to keep plots and modify by hand as smilar as https://github.com/scikit-hep/cabinetry/discussions/425?

**Comment by alexander-held** [comment, marked as answer]
There is no option for this, but this is straightforward to do externally. Given some list `figs` returned from `visualize.data_mc`, you can change the range for a specific figure via something like
```python
figs[0]["figure"].axes[1].set_ylim([0.9,1.1])
figs[0]["figure"].savefig("figure_name.pdf")
```
where the `[1]` selects the ratio panel. You might also want to use `set_yticks` / `set_yticklabels` to update ticks and tick labels in this case.

**Reply by Tomoya-Iizawa** [reply]
Thank you! It works!
